evennia/contrib/ingame_python/scripts.py

```diff
@@ -101,7 +101,7 @@ def get_events(self, obj):
         Return a dictionary of events on this object.
 
         Args:
-            obj (Object): the connected object.
+            obj (Object or typeclass): the connected object or typeclass.
```

Comment:
This seems inaccurate. Only the connected object or an ObjectDB instance is allowed as-is.